--- medusa-0.5.1/libmedusa/medusa-file-info-utilities.h~	Thu Dec 21 14:28:44 2000
+++ medusa-0.5.1/libmedusa/medusa-file-info-utilities.h	Mon Dec  3 01:38:06 2001
@@ -31,6 +31,7 @@
 
 #include <glib.h>
 #include <time.h>
+#include <sys/types.h>
 
 /* FIXME bugzilla.eazel.com 2998: 
    These should not have the "file_info" name tagged on */
--- medusa-0.5.1/indexer/Makefile.am~	Tue Feb 13 01:38:32 2001
+++ medusa-0.5.1/indexer/Makefile.am	Mon Dec  3 01:50:05 2001
@@ -14,6 +14,7 @@
 	-I$(top_srcdir) \
 	-I$(top_builddir) \
 	$(GLIB_CFLAGS) \
+	$(GNOME_CFLAGS) \
 	$(XML_CFLAGS) \
 	$(VFS_CFLAGS) \
 	$(NULL)
--- medusa-0.5.1/indexer/Makefile.in~	Fri May  4 08:08:30 2001
+++ medusa-0.5.1/indexer/Makefile.in	Mon Dec  3 02:05:34 2001
@@ -133,7 +133,7 @@
 DEFS =  	$(NULL)
 
 
-INCLUDES =  	-I$(top_srcdir) 	-I$(top_builddir) 	$(GLIB_CFLAGS) 	$(XML_CFLAGS) 	$(VFS_CFLAGS) 	$(NULL)
+INCLUDES =  	-I$(top_srcdir) 	-I$(top_builddir) 	$(GLIB_CFLAGS) 	$(GNOME_CFLAGS) 	$(XML_CFLAGS) 	$(VFS_CFLAGS) 	$(NULL)
 
 
 medusa_indexd_LDADD =  	$(top_builddir)/libmedusa/libmedusa.la 	$(top_builddir)/libmedusa-internal/libmedusa-index.la 	$(VFS_LIBS) 	$(GLIB_LIBS) 	$(GNOME_LIBS) 	$(OAF_LIBS) 	$(XML_LIBS) 	$(NULL)
--- medusa-0.5.1/search-service/Makefile.am~	Wed Oct  4 20:36:38 2000
+++ medusa-0.5.1/search-service/Makefile.am	Mon Dec  3 02:10:28 2001
@@ -18,6 +18,7 @@
 	-I$(top_srcdir)/libmedusa-internal \
 	-I$(top_srcdir)/libmedusa \
 	$(GLIB_CFLAGS) \
+	$(GNOME_CFLAGS) \
 	$(VFS_CFLAGS) \
 	$(NULL)
 
--- medusa-0.5.1/search-service/Makefile.in~	Fri May  4 08:08:31 2001
+++ medusa-0.5.1/search-service/Makefile.in	Mon Dec  3 02:10:55 2001
@@ -133,7 +133,7 @@
 noinst_HEADERS =  	medusa-authenticate.h 	medusa-file-search-parse-transmission.h 	$(NULL)
 
 
-INCLUDES =  	-I$(top_srcdir) 	-I$(top_srcdir)/libmedusa-internal 	-I$(top_srcdir)/libmedusa 	$(GLIB_CFLAGS) 	$(VFS_CFLAGS) 	$(NULL)
+INCLUDES =  	-I$(top_srcdir) 	-I$(top_srcdir)/libmedusa-internal 	-I$(top_srcdir)/libmedusa 	$(GLIB_CFLAGS) 	$(GNOME_CFLAGS) 	$(VFS_CFLAGS) 	$(NULL)
 
 
 medusa_searchd_LDADD =  	$(top_builddir)/libmedusa/libmedusa.la 	$(top_builddir)/libmedusa-internal/libmedusa-index.la 	$(VFS_LIBS) 	$(GLIB_LIBS) 	$(GNOME_LIBS) 	$(OAF_LIBS) 	$(XML_LIBS) 	$(NULL)
--- medusa-0.5.1/tests/Makefile.am~	Sat Apr  7 02:47:07 2001
+++ medusa-0.5.1/tests/Makefile.am	Mon Dec  3 02:16:16 2001
@@ -9,6 +9,7 @@
 	$(VFS_CFLAGS) \
 	$(GLIB_CFLAGS) \
 	$(GTK_CFLAGS) \
+	$(GNOME_CFLAGS) \
 	$(NULL)
 
 test_text_index_SOURCES = \
--- medusa-0.5.1/tests/Makefile.in~	Fri May  4 08:08:33 2001
+++ medusa-0.5.1/tests/Makefile.in	Mon Dec  3 02:16:45 2001
@@ -127,7 +127,7 @@
 
 noinst_PROGRAMS = test-text-index test-index-service test-queue test-file-indexer test-versioned-file test-search-method test-system-state
 
-INCLUDES =  	-I$(top_srcdir) 	-I$(top_srcdir)/libmedusa-internal 	-I$(top_srcdir)/libmedusa 	$(VFS_CFLAGS) 	$(GLIB_CFLAGS) 	$(GTK_CFLAGS) 	$(NULL)
+INCLUDES =  	-I$(top_srcdir) 	-I$(top_srcdir)/libmedusa-internal 	-I$(top_srcdir)/libmedusa 	$(VFS_CFLAGS) 	$(GLIB_CFLAGS) 	$(GTK_CFLAGS) 	$(GNOME_CFLAGS) 	$(NULL)
 
 
 test_text_index_SOURCES =  	medusa-text-index-test.c
