--- medusa-0.5.1/configure.in~	Sun Feb 17 01:45:53 2002
+++ medusa-0.5.1/configure.in	Sun Feb 17 01:45:53 2002
@@ -56,9 +56,9 @@
   )
  )
 else
- AC_CHECK_LIB(db, dbopen, DB_LIB="-ldb",
+ AC_CHECK_LIB(db-3, __db185_open, DB_LIB="-ldb-3",
   AC_CHECK_LIB(db1, dbopen, DB_LIB="-ldb1",
-   AC_CHECK_LIB(db-3, __db185_open, DB_LIB="-ldb-3",
+   AC_CHECK_LIB(db, dbopen, DB_LIB="-ldb",
     AC_MSG_ERROR([Your db library is missing db 1.85 compatibility mode])
    )
   )
